src/components/Users/CreateUserForm.tsx

```diff
@@ -365,9 +403,15 @@ export default function CreateUserForm({ onSubmitSuccess }: Props) {
             name="date_of_birth"
             render={({ field }) => (
               <FormItem>
-                <FormLabel>{t("date_of_birth")}</FormLabel>
+                <FormLabel required>{t("date_of_birth")}</FormLabel>
                 <FormControl>
-                  <Input type="date" {...field} />
+                  <DatePicker
+                    date={field.value}
+                    onChange={(date) => field.onChange(date)}
+                    disabled={(date) =>
+                      date > new Date() || date < new Date("1900-01-01")
+                    }
+                  />
```

Comment:
Why was this changed to `DatePicker`? Try entering a realistic date of birth. It's really hard for the users to click thousand times to reach the desired month